style.css

```diff
@@ -574,25 +575,26 @@ article h1 {
 form {
   margin-top: -30px;
   display: flex;
-  margin-right: 9%;
+  margin-right: 15%;
+  padding: 24px;
 }
 
 .footer li input {
   border: 1px solid #cfd8dc;
   height: 48px;
-  width: 312px;
+  width: 300px;
   margin-top: 10px;
   border-radius: 8px;
-  padding: 15px;
+  /* padding: 24px; */
```

Comment:
- [ ] Good job reducing the padding as suggested by the previous reviewer :clap:, kindly make sure to remove it completely a line of code if necessary instead of commenting it out. :recycle: